sqlglot/tokens.py

```diff
@@ -492,6 +495,7 @@ class Tokenizer(metaclass=_Tokenizer):
         "PARTITION": TokenType.PARTITION,
         "PARTITION BY": TokenType.PARTITION_BY,
         "PARTITIONED BY": TokenType.PARTITIONED_BY,
+        "PARTITIONED_BY": TokenType.PARTITIONED_BY,
```

Comment:
now that you've added this property, we'll need to add partitoned_by to the id_vars const